pom.xml

```
@@ -35,6 +35,7 @@
 		<module>api</module>
 		<module>api-metadata</module>
 		<module>omod</module>
+		<module>fhir</module>
```

OMOD is the final artifact, so it always goes last.

```suggestion
		<module>fhir</module>
		<module>omod</module>
```